application-chartjs-ui/src/main/resources/XWiki/ChartJS/WebHome.xml

```diff
@@ -247,6 +246,40 @@ require(['jquery', 'chart', 'datalabels', 'palette'], function($, Chart, datalab
     return 'rgba(' + r + ',' + g + ',' + b + ',' + opacity + ')';
   };
 
+  // Removes 0 values from the datasets, from the corresponding colors and from the labels.
+  var removeZeros = function(data, options, type) {
+    if (options.removeZeros &amp;&amp; data.datasets.length == 1) {
+      var newLabels = [];
+
+      data.datasets.forEach(function(dataSet, index) {
```

Comment:
It's a bit strange that you need to do ``forEach`` when you know there's only one data set.